.github/workflows/unittest.yaml

```diff
@@ -0,0 +1,36 @@
+name: unittest
+
+on:
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  get-json-config:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          cd bin
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install pytest pytest-html jsonschema
+
+      - name: Run unit tests
+        run: |
+          cd bin
+          source .venv/bin/activate
+          # TODO: rename files with dash that are used as modules e.g. get-json-config.py
+          # Workaround is to create a symlink to the file or use __import__("foo-bar")
```

Comment:
here is the explanation for the symlink: once rickshaw is changed to call the script without dashes, we can get rid of the symlink and rename the file.